packages/node_modules/@webex/internal-plugin-calendar/test/integration/spec/calendar.js

```diff
@@ -90,9 +90,53 @@ function postToWhistler(webex, type, payload) {
     }));
 }
 
-describe.skip('plugin-calendar', () => {
+function postNotesToWhistler(webex, type, payload) {
+  return retry(() => webex.request({
+    method: 'POST',
+    service: 'whistler',
+    resource: 'calendarEvent',
+    body: payload,
+    qs: {
+      changeType: type,
+      useProduction: typeof process.env.WDM_SERVICE_URL === 'undefined' || process.env.WDM_SERVICE_URL.includes('wdm-a.wbx2.com')
+    }
+  })
+    .then((res) => {
+      // Test response is using our parameters
+      const createdMeeting = res.body;
+
+      assert.isDefined(createdMeeting);
+      assert.equal(createdMeeting.meeting.meetingNotes, payload.meetingNotes);
+
+      return res;
+    }));
+}
+
+function postParticipantsToWhistler(webex, type, payload) {
+  return retry(() => webex.request({
+    method: 'POST',
+    service: 'whistler',
+    resource: 'calendarEvent',
+    body: payload,
+    qs: {
+      changeType: type,
+      useProduction: typeof process.env.WDM_SERVICE_URL === 'undefined' || process.env.WDM_SERVICE_URL.includes('wdm-a.wbx2.com')
+    }
+  })
+    .then((res) => {
+      // Test response is using our parameters
+      const createdMeeting = res.body;
+
+      assert.isDefined(createdMeeting);
+      assert.equal(createdMeeting.meeting.itemId, payload.id);
+
+      return res;
+    }));
+}
+
+describe('plugin-calendar', () => {
   describe('Calendar', () => {
-    describe('#list()', function () {
+    describe.skip('#list()', function () {
```

Comment:
Why are you skipping these tests?

Reply:
There was only 1 test in this file earlier, and it was inside a `describe.skip()`, so I moved the `.skip()` on the inside to that test, added my tests to the outer describe block and removed the `.skip()` from there (line 93)